scripts/local-3node.sh

```
@@ -0,0 +1,276 @@
+  screen_list="$(screen -ls 2>/dev/null || true)"
+  if printf '%s\n' "${screen_list}" | rg -q "${SESSION}" || ps -ax -o command= 2>/dev/null | rg -q "SCREEN .*${SESSION}"; then
+    echo "  ${SESSION} (running)"
+  else
+    echo "  ${SESSION} (not running)"
+  fi
+
+  echo "Processes:"
+  local procs
+  procs="$(ps -ax -o pid= -o command= 2>/dev/null | rg "${APP_PATTERN}" | rg -v 'rg |ps -ax|/bin/zsh -c' || true)"
+  if [[ -n "${procs}" ]]; then
+    echo "${procs}" | sed 's/^/  /'
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)